js/src/utils/openSeadragon.js

```diff
@@ -1,22 +1,70 @@
-(function($) {
+(function ($) {
 
-  $.OpenSeadragon = function(options) {
+  $.OpenSeadragon = function (options) {
 
     var osd = OpenSeadragon(
-
       jQuery.extend({
         preserveViewport: true,
-        visibilityRatio:  1,
-        minZoomLevel:     0,
+        visibilityRatio: 1,
+        minZoomLevel: 0,
         defaultZoomLevel: 0,
-        blendTime:        0.1,
-        alwaysBlend:      false,
-        prefixUrl:        'images/openseadragon/',
+        blendTime: 0.1,
+        alwaysBlend: false,
+        prefixUrl: 'images/openseadragon/',
```

Comment:
I think this setting is no longer needed. Can we not make white space changes so that a00ca942764dd670ec0e86ed4c11affaba4db9a8 will still work with this?